src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;

```

Comment:
The import statement for 'java.util.Random' is unnecessary as there are no usages of Random in the Ball class. It's good practice to remove unused imports to keep the code clean.